packages/blade/src/components/Button/BaseButton/__tests__/BaseButton.web.test.tsx

```diff
@@ -152,228 +164,31 @@ describe('<BaseButton />', () => {
-    const buttonText = 'Pay Now';
-    const { container } = renderWithTheme(
-      <BaseButton intent="information" contrast="low" isDisabled={true}>
-        {buttonText}
-      </BaseButton>,
-    );
-    expect(container).toMatchSnapshot();
-  });
-
-  it('should render positive intent high contrast button', () => {
-    const buttonText = 'Pay Now';
-    const { container } = renderWithTheme(
-      <BaseButton intent="positive" contrast="high">
-        {buttonText}
-      </BaseButton>,
-    );
-    expect(container).toMatchSnapshot();
-  });
-  it('should render disabled positive intent high contrast button', () => {
-    const buttonText = 'Pay Now';
-    const { container } = renderWithTheme(
-      <BaseButton intent="positive" contrast="high" isDisabled={true}>
-        {buttonText}
-      </BaseButton>,
-    );
-    expect(container).toMatchSnapshot();
-  });
-
-  it('should render negative intent high contrast button', () => {
-    const buttonText = 'Pay Now';
-    const { container } = renderWithTheme(
-      <BaseButton intent="negative" contrast="high">
-        {buttonText}
-      </BaseButton>,
-    );
-    expect(container).toMatchSnapshot();
-  });
-  it('should render disabled negative intent high contrast button', () => {
-    const buttonText = 'Pay Now';
-    const { container } = renderWithTheme(
-      <BaseButton intent="negative" contrast="high" isDisabled={true}>
-        {buttonText}
-      </BaseButton>,
-    );
-    expect(container).toMatchSnapshot();
-  });
-
-  it('should render neutral intent high contrast button', () => {
-    const buttonText = 'Pay Now';
-    const { container } = renderWithTheme(
-      <BaseButton intent="neutral" contrast="high">
-        {buttonText}
-      </BaseButton>,
-    );
-    expect(container).toMatchSnapshot();
-  });
-  it('should render disabled neutral intent high contrast button', () => {
-    const buttonText = 'Pay Now';
-    const { container } = renderWithTheme(
-      <BaseButton intent="neutral" contrast="high" isDisabled={true}>
-        {buttonText}
-      </BaseButton>,
-    );
-    expect(container).toMatchSnapshot();
-  });
-
-  it('should render notice intent high contrast button', () => {
-    const buttonText = 'Pay Now';
-    const { container } = renderWithTheme(
-      <BaseButton intent="notice" contrast="high">
-        {buttonText}
-      </BaseButton>,
-    );
-    expect(container).toMatchSnapshot();
-  });
-  it('should render disabled notice intent high contrast button', () => {
-    const buttonText = 'Pay Now';
-    const { container } = renderWithTheme(
-      <BaseButton intent="notice" contrast="high" isDisabled={true}>
-        {buttonText}
-      </BaseButton>,
-    );
-    expect(container).toMatchSnapshot();
-  });
+  colors.forEach((color) => {
+    variants.forEach((variant) => {
+      // We support only white and default color for tertiary variant
+      if (variant === 'tertiary' && color !== 'white' && color !== 'default') return;
```

Comment:
can we add an assertion which throws error when someone passes the tertiary for feedback colors?